porting/examples/dummy/Makefile

```diff
@@ -17,7 +17,7 @@ INC = \
 
 OBJ := $(SRC:.c=.o)
 
-CFLAGS := $(NIMBLE_CFLAGS) -DBLE_NPL_STORAGE_DECL=
+CFLAGS := $(NIMBLE_CFLAGS)
```

Comment:
no need to change this, it was on purpose as we default to `static inline` declarations

Reply:
But then when the define is missing, NPL using simple function calls fails to link...
Better is to default to a working, simple solution and then add defines to enable optimizations.